aeneas/src/ir/Normalization.v3

```diff
@@ -653,7 +653,6 @@ class ReachabilityNormalizer(config: NormalizerConfig, ra: ReachabilityAnalyzer)
 			if (specializer.normCode(context, rm)) return;
 		}
 		SsaRaNormalizer.new(context, this).build(rm.norm);
-		if (rm.norm.source != null) rm.norm.source.irIndex = newIr.methods.length;
```

Comment:
I'm not sure why this line was removed. Can you explain?

Reply:
I add this field `irIndex` before to try to retrieve irMethod from vstMethod in debugger. It is not used anymore